dbms/src/Flash/Mpp/FineGrainedShuffleWriter.h

```diff
@@ -19,18 +19,15 @@
 #include <Flash/Mpp/TrackedMppDataPacket.h>
 #include <common/types.h>
 
-namespace DB::HashBaseWriterHelper
-{
-struct HashPartitionWriterHelperV1;
-}
-
```

Comment:
why remove this?

Reply:
I didn't see anyone use this struct.  So i think it's useless